homeassistant/components/history/__init__.py

```diff
@@ -700,6 +700,8 @@ def __init__(self, row):  # pylint: disable=super-init-not-called
         self._row = row
         self.entity_id = self._row.entity_id
         self.state = self._row.state
+        if self.state is None:
```

Comment:
`self.state = self._row.state if self._row.state not None else ""` ?

Reply:
No, avoid if-else if possible. Always.